app/controllers/budgets/investments_controller.rb

```diff
@@ -32,13 +34,18 @@ class InvestmentsController < ApplicationController
     respond_to :html, :js
 
     def index
+      all_investments = investments
```

Comment:
I wonder if it's possible to simplify the code a bit :thinking:. How about something like:

```ruby
all_investments = if @budget.finished? 
                    investments.winners
                  else
                    investments
                  end

@investments = all_investments.page(params[:page]).per(10).for_render
# ...
```